ocs_ci/framework/conf/flexy_default_config.yaml

```diff
@@ -0,0 +1,15 @@
+ENV_DATA:
+  platform: 'baremetalpsi'
+  deployment_type: 'upi'
+  flexy_img_url: 'docker-registry.upshift.redhat.com/aosqe/flexy:poc'
+
+# Override flexy params here
+FLEXY:
+  default_private_conf_repo: 'git@gitlab.cee.redhat.com:ocs/flexy-ocs-private.git'
+  flexy_private_conf_url: ''
+  # Volume which will be attached to instance,
+  # which will be consumed for osd. (size in GB)
+  volume_size: 500
```

Comment:
Wouldn't be better/sufficient to use existing `device_size` configuration variable?

e.g. https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocsci/production_device_size.yaml#L4